packages/cli/generators/app/templates/src/repositories/README.md

```diff
@@ -0,0 +1,3 @@
+# Repositories
```

Comment:
The `Mixin`, `Providers`, etc.. pages have a great deal of description. Can we add some links to our docs or brief explanations here as well?

Reply:
The READMEs are copied from https://github.com/strongloop/loopback4-extension-starter. @virkt25 Do you make the docs for Mixin/Provider/Decorator available to http://loopback.io/doc/en/lb4/Concepts.html? We should only keep links in the README.md to be scaffolded into user projects.